src/rachanaGJun20/PlindromeString.java

```diff
@@ -0,0 +1,41 @@
+package rachanaGJun20;
+
+import java.util.Scanner;
+
+public class PlindromeString {
+	//method will check whether given string is Palindrome
+	boolean isStringPlindrome(String str) {
+		String revstr = "";
+		String givenString = str;
```

Comment:
Suggestion 1: Instead of assigning value to new string , we can use directly Str value(In line 14) as well

Reply:
Thanks Ajit , Yes got it will make that change